spec/controllers/verify/confirmations_controller_spec.rb

```diff
@@ -86,6 +86,13 @@ def stub_idv_session
 
         get :index
       end
+
+      it 'creates an `account_verified` event once per confirmation' do
+        get :index
+        user.reload
+
+        expect(user.events.where(event_type: 8).size).to be 1
```

Comment:
Cool, I saw in the documentation that you can call `ClassName.my_enum` but didn't make the leap to trying it with an instance. Thanks!